bin/docker_push

```diff
@@ -2,15 +2,22 @@
 
 set -evuo pipefail
 
-IMAGE_NAME=${TRAVIS_REPO_SLUG:-${USER}/skipper}
+IMAGE_NAME=$DOCKER_REPO
 DOCKER_IMAGE="${IMAGE_NAME}:${VERSION/+/-}"
 
 if [[ "${CI:-}" == "true" ]]; then
-  # Run multi-arch build and push it
-  docker buildx build --pull \
-    --tag "${IMAGE_NAME}:latest" \
-    --tag "${DOCKER_IMAGE}" \
-    --platform=linux/amd64 \
-    --push .
-    echo "Pushed image ${DOCKER_IMAGE}"
```

Comment:
It might be useful to keep the "pushed image $DOCKER_IMAGE" - I think docker push doesn't (or didn't?) output the tag(s) it pushes